test/native/pom.xml

```diff
@@ -45,6 +45,18 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-cluster-mode-repository-etcd</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.vertx</groupId>
+            <artifactId>vertx-grpc</artifactId>
+            <version>${vertx.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
- @RaigorJiang Is there any background in #29747 ? I don't understand why `org.apache.shardingsphere:shardingsphere-cluster-mode-repository-etcd` needs to exclude the dependency of `io.vertx:vertx-grpc`.